test/test-v8debugapi.ts

```diff
@@ -946,10 +986,12 @@ describe('v8debugapi', function() {
             assert.equal(((fooVal as any).members as any).length, 4);
             assert.strictEqual((foo as any).status, undefined);
 
-            api.clear(bp);
             config.capture.maxDataSize = oldMaxData;
```

Comment:
The same concern about moving these lines inside the callback of `api.clear` as described above.